python/pyarrow/tests/test_dataset.py

```diff
@@ -682,18 +685,12 @@ def test_open_dataset_unsupported_format(tempdir):
         ds.dataset([path], format="blabla")
 
 
-@pytest.mark.parquet
-def test_open_dataset_from_source_additional_kwargs(tempdir):
```

Comment:
Needs restoring, also handled in https://github.com/bkietz/arrow/pull/1/files#diff-ebae6bdb86cce330cd1a70ff88cd8c19R711

Reply:
I'll re add that test, thanks